webapp/src/components/forms/bulk_add_channel_form.tsx

```diff
@@ -209,7 +209,7 @@ const ActualForm = (props: ActualFormProps) => {
             required: false,
             helpText: (
                 <div>
-                    {'Add guests if they are present on the file. If this is unchecked guests wont be added to the team if the above setting is not checked.'}
+                    {'Add guests if they are present on the file. If this is unchecked guests wont be added to the team.'}
```

Comment:
> added to the team

Is this about being added to the team or the channel? This setting must be toggled for a guest to be added to the channel, even if they are already a guest in another channel of the same team. Being added to a team or not is handled by the previous setting. 

To update the above and to follow the wording style of the main product:

```suggestion
                    {'Adds guests, if they are present on the file. If unchecked, guests will not be added to the channel.'}
```